search-app/src/cmr/search/api/routes.clj

```diff
@@ -365,6 +376,11 @@
                {params :params headers :headers ctx :request-context}
                (get-deleted-collections ctx path-w-extension params headers)))
 
+        (context ["/:path-w-extension" :path-w-extension #"(?:collections-created-after)(?:\..+)?"] [path-w-extension]
```

Comment:
This is now treated as a standard date-range parameter at the `search/collections/` endpoint